### PR TITLE
Update metabuild deps so pip check is working correctly

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* ~[ ] Reset the build number to `0` (if the version changed)~
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by goanpeca** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,6 +29,7 @@ test:
   imports:
     - napari_matplotlib
   commands:
+    - "python -c 'from napari_matplotlib._version import version;print(version)'"
```
Comment (dstansby) [inline recipe/meta.yaml]:
```suggestion
    - "python -c 'from napari_matplotlib import __version__;print(__version__)'"
```
So it's testing public API?

Comment (goanpeca) [reply recipe/meta.yaml]: Yep that also works, thanks :) 

When using `pip list` I was getting a 0.0.0 version which is affecting another pip check on a different PR that depended on this one.

Comment (goanpeca) [reply recipe/meta.yaml]: See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=518824&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2651

I missed that :)

**Pull request merged**